_targets.R

```diff
@@ -117,24 +117,24 @@ targets_list <- list(
 
   # BUILD QUERY
   # Define list of GCMs
-  tar_target(gcm_names, c('ACCESS', 'GFDL', 'CNRM', 'IPSL', 'MRI', 'MIROC')),
+  tar_target(gcm_names, c('ACCESS', 'GFDL', 'CNRM', 'IPSL', 'MRI', 'MIROC5')),
   tar_target(gcm_dates_df,
              tibble(
-               projection_period = c('1980_1999', '2040_2059', '2080_2099'),
-               start_datetime = c('1980-01-01 00:00:00', '2040-01-01 00:00:00', '2080-01-01 00:00:00'),
+               projection_period = c('1980_2099'),
+               start_datetime = c('1980-01-01 00:00:00'),
                # Include midnight on the final day of the time period
-               end_datetime = c('2000-01-01 00:00:00', '2060-01-01 00:00:00', '2100-01-01 00:00:00')
+               end_datetime = c('2100-01-01 00:00:00')
```

Comment:
Does it actually go to 2100 or is the last date in 2099?

Reply:
I am seeing max date is 2099-12-30 (but expect it to be 2099-12-31. I suppose I could write this as `2099-12-31 23:59:59` instead.